custom/icds_reports/reports/service_delivery_dashboard_data.py

```diff
@@ -80,7 +80,7 @@ def should_show_25():
     def update_total_row(first_dict, second_dict):
         for key, value in first_dict.items():
             # excluding location and percentage fields
-            if key not in location_fields + ['thr', 'pse', 'sn', 'gm', 'home_visits']:
+            if key not in location_fields + ['cbe', 'thr', 'pse', 'sn', 'gm', 'home_visits']:
```

Comment:
it will be good to add the test data to test this change fixes the issue

Reply:
added